docs/style.css

```diff
@@ -255,16 +252,16 @@ li {
 	border-width: 0;
 }
 
-#cgit > .content {
-	margin: 1rem 0;
-}
-
 #cgit > .footer {
 	display: flex;
 	justify-content: center;
 	color: #ccc;
 }
 
+#cgit .footer a:first-child {
+	margin-left: .5em;
+}
+
 	/* remove border between columns in header & tabs */
 #cgit > table.tabs > tbody > tr > td , #cgit > table#header > tbody > tr > td {
```

Comment:
<img width="365" alt="Screenshot 2024-01-25 at 23 16 17" src="https://github.com/underground-software/orbit/assets/1909841/dc2ab1e4-ec73-4665-a923-be76bf2ae4f5">

A bit cozy don't you think?